src/V1Meta.js

```diff
@@ -54,4 +54,10 @@ export default class V1Meta {
         const headers = createHeaderObj(this.authHeader);
         return Promise.resolve(this.postFn(url, null, headers));
     }
+
+    queryDefinition(assetType) {
+        const url = `${this.urls.meta()}/${assetType}`;
+        const headers = createHeaderObj(this.authHeader);
+        return Promise.resolve(this.postFn(url, null, headers));
```

Comment:
If you use an `Accept` header or an `?accept=` query parameter you can get XML or JSON.